persistence/src/main/java/org/mskcc/cmo/metadb/persistence/MetaDbSampleRepository.java

```diff
@@ -73,7 +73,7 @@ MetaDbSample findMetaDbSampleByRequestAndIgoId(@Param("reqId") String reqId,
     )
     List<SampleMetadata> findSampleMetadataListByCmoPatientId(@Param("cmoPatientId") String cmoPatientId);
     
-    @Query("MATCH (sm:SampleMetadata {cmoInfoIgoId: $igoSampleId})"
+    @Query("MATCH (sm:SampleMetadata {igoSampleId: $igoSampleId})"
             + "RETURN sm")
-    List<SampleMetadata> getSampleMetadataHistory(@Param("igoSampleId") String igoSampleId);
+    List<SampleMetadata> getSampleMetadataHistoryByIgoSampleId(@Param("igoSampleId") String igoSampleId);
```

Comment:
`getSampleMetadataHistoryByIgoSampleId` --> `getSampleMetadataHistoryByIgoId`

and change param to `@Param("igoSampleId") String igoSampleId` --> `@Param("igoId") String igoId`